src/routes/polls/polls.controller.js

```diff
@@ -1,11 +1,32 @@
 const db = require("./../../db/index");
 
-const postPolls = function (req, res) {
+const postPolls = function(req, res) {
   const body = req.body;
   console.log(body);
-  res.set('Content-Type', 'application/json')
+  res.set("Content-Type", "application/json");
   res.send(`Received new Poll: ${body.message}`);
-}
+
+  // Function to insert data
+  const insertData = (col, data) => {
```

Comment:
This function should probably be defined outside the `postPolls` function so that it can be reused.

Reply:
👍Moved this out of the function :)